README.md

```diff
@@ -1 +1,108 @@
-# Capstone_project
\ No newline at end of file
+
+<h3 id = "readme-top"><b>Capstone Project </b></h3>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+- [📖 Title](#Team Event)
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 Live Demo from loom](#Live-Loom-demo)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Credit](#Credit)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project] <a name="about-project">https://github.com/sam2if/Capstone_project.git</a>
+
+
+
+# Team Event
+This Capstone Project is based on an online website for a conference with a slogan "Reduce, Reuse and Recycle". 
+ You will learn how you can create enterprise value, empower greater worker productivity, and make the world a better place through advanced technology. 
+
+## Built With
+
+- Html
+- Css
+- JavaScript
+
+### Key Features <a name="key-features">
+- **[key_feature_1]** Capstone Project: built with CSS, Html and JavaScript.
+- **[key_feature_2]** Capstone Project: Mobile Version.
+- **[key_feature_3]** Capstone Project: The Home me page.
+- **[key_feature_4]** Capstone Project: desktop version.
+- **[key_feature_5]** Capstone Project: About me page for both mobile and desktop version.
+
+
+### Home page
+
+
+## Live Demo
+
+[Live Demo Link](https://sam2if.github.io/)
+
+## Live Loom demo
+
+[Live Demo from loom ](https://www.loom.com/share/e5a70bde65564b94be28ef1b86d5efb8)
```

Comment:
- [ ] @sam2if great work on your presentation, however, your presentation is more than the required 5 minutes or less, kindly ensure your video is not more than 5 minutes, also the audio was not that clear it would be nice if it's better.
![image](https://user-images.githubusercontent.com/61408860/225848437-90b1231b-9a5c-4f45-a587-92360061a487.png)
kindly ensure your presentation follows all the guidelines shown below 
![image](https://user-images.githubusercontent.com/61408860/225848720-07a3734b-c960-475f-915f-be9d8799104a.png)